frontend/projects/upgrade/src/app/features/dashboard/experiments/pages/experiment-details-page/experiment-details-page-content/experiment-exclusions-section-card/experiment-exclusions-section-card.component.html

```diff
@@ -25,5 +25,7 @@
   <app-experiment-exclusions-table
     content
     *ngIf="isSectionCardExpanded"
+    [actionsDisabled]="!(permissions$ | async)?.experiments.update"
```

Comment:
Why do we need this attribute for the exclusions table but not the inclusions table?

Reply:
Good catch! I've updated the inclusions table to disable the slide toggle and action buttons when update permission is not allowed.